packages/client-twitter/src/interactions.ts

```diff
@@ -500,7 +500,12 @@ export class TwitterInteractionClient {
                     }
 
                     await this.runtime.processActions(
-                        message,
+                        {...message,
```

Comment:
this is good.

Reply:
after this change, we could get more tweet info(like tweet id and auther username) from action handler, for example:
https://github.dev/degencast/eliza/blob/4130f605ebe8d966ac1a18847bb8f3a773218e5b/packages/plugin-degencast/src/actions/airdrop.ts#L57-L59